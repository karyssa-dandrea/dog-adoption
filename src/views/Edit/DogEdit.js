import React from 'react';

export default function DogEdit() {
  return (
    <div>
      <h1>Edit Dog</h1>
    </div>
  );
}
