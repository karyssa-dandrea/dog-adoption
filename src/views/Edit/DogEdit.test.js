import { render, screen } from '@testing-library/react';
import DogEdit from '../../views/Edit/DogEdit';
import { MemoryRouter, Route } from 'react-router-dom';

test('should render dog details in a form', async () => {
  const { container } = render(
    <MemoryRouter initialEntries={['/dogs/4/edit']}>
      <Route path="/dogs/:id/edit" component={DogEdit} />
    </MemoryRouter>
  );
  await screen.findByDisplayValue('Maggie');
  expect(container).toMatchSnapshot();
});
