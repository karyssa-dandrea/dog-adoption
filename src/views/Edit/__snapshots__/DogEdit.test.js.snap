// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render dog details in a form 1`] = `
<div>
  <div>
    <h1>
      Edit Dog
    </h1>
    <div
      class="header"
    >
      <header>
        <a
          aria-current="page"
          class="active"
          href="/dogs"
        >
           Home 
        </a>
        <a
          href="/admin"
        >
           Admin
        </a>
      </header>
    </div>
    <div
      class="form"
    >
      <h2>
        Add Dog Details
      </h2>
      <p />
      <div
        class="dog-form"
      >
        <label>
          Name
        </label>
        <input
          placeholder="name"
          type="text"
          value="Maggie"
        />
        <label>
          Bio:
        </label>
        <input
          placeholder="bio"
          type="text"
          value="Too cute to care!"
        />
        <label>
          Age:
        </label>
        <input
          placeholder="age"
          type="text"
          value="2 months"
        />
        <label>
          Image:
        </label>
        <input
          placeholder="image"
          type="text"
          value="https://placedog.net/samples/178.jpg"
        />
        <label>
          Breed:
        </label>
        <input
          placeholder="breed"
          type="text"
          value="Australian Shepard"
        />
      </div>
      <div
        class="button"
      >
        <button>
          Save
        </button>
      </div>
    </div>
  </div>
</div>
`;
